tests/catboost/test_CatBoost_converter.py

```diff
@@ -0,0 +1,53 @@
+"""
+Tests for CatBoostRegressor and CatBoostClassifier converter.
+"""
+import unittest
+import numpy
+import catboost
+from sklearn.datasets import make_regression, make_classification
+from onnxmltools.convert import convert_catboost
+from onnxmltools.utils import dump_data_and_model, dump_single_regression, dump_multiple_classification
+
+
+class TestCatBoost(unittest.TestCase):
+    def test_catboost_regressor(self):
+        X, y = make_regression(n_samples=100, n_features=4, random_state=0)
+        catboost_model = catboost.CatBoostRegressor(task_type='CPU', loss_function='RMSE',
+                                                    n_estimators=10, verbose=0)
+        dump_single_regression(catboost_model)
+
+        catboost_model.fit(X.astype(numpy.float32), y)
+        catboost_onnx = convert_catboost(catboost_model, name='CatBoostRegression',
+                                         doc_string='test regression')
+        self.assertTrue(catboost_onnx is not None)
+        dump_data_and_model(X.astype(numpy.float32), catboost_model, catboost_onnx, basename="CatBoostReg-Dec4")
+
+    def test_catboost_bin_classifier(self):
+        X, y = make_classification(n_samples=100, n_features=4, random_state=0)
+        catboost_model = catboost.CatBoostClassifier(task_type='CPU', loss_function='CrossEntropy',
+                                                     n_estimators=10, verbose=0)
+
+        catboost_model.fit(X.astype(numpy.float32), y)
+
+        catboost_onnx = convert_catboost(catboost_model, name='CatBoostBinClassification',
+                                         doc_string='test binary classification')
+        self.assertTrue(catboost_onnx is not None)
+        # onnx runtime returns zeros as class labels
+        # dump_data_and_model(X.astype(numpy.float32), catboost_model, catboost_onnx, basename="CatBoostBinClass")
```

Comment:
Should this line be uncommented?

Reply:
Well, this part has a problem :( 
The comparison works properly with probabilities, not with labels. A converted model returns only zeros as labels.  I consulted the Catboost team and they consider it as onnxruntime bug.

Reply:
This must be fixed and it is probably an error somewhere in the onnx graph.

Reply:
Thank you for the information, I reported your reply to the Catboost team members and I will update my pr after they fix the bug

Reply:
It works now with the new onnxruntime version